forch/http_server.py

```diff
@@ -129,7 +129,12 @@ def _split_request(self, base_path, req_path):
         full_path = os.path.join(self._root_path, path)
         if os.path.isdir(full_path):
             full_path = os.path.join(full_path, self._DEFAULT_FILE)
-        return self.read_file(full_path)
+        try:
+            content = self.read_file(full_path)
+            return content
+        except Exception as error:
+            LOGGER.warning("Received illegal request: %s", req_path)
+            return str(error)
```

Comment:
what does this return to the requesting agent (web browser)?  Is it just a string-but-success-code, or a proper 400* or 500* code?

Reply:
Returning appropriate http code now